### DISCUSSION TITLE
v1.8.10: Medicine calendar view

### DISCUSSION THREAD

**Discussion by Futsch1** [Futsch1/medTimer#168] [Announcements]
WIth v1.8.10, a new button with a calendar icon was added to the medicine view. It opens a screen that shows a calendar of the past and future 30 days. When selected, either the previous reminders triggered for that day or the future reminders are shown. This can be used to get a brief idea about past doses and about the effect of advanced reminder settings.

<img src='https://github.com/Futsch1/medTimer/assets/63090558/33382716-1a60-47a9-a2b7-02bd3f8555ee)' height=400 />

**Comment by NyaomiDEV** [comment]
I wonder if you could do the same on the analysis view, listing days in a calendar and showing taken / skipped medications that way (as if those were events on a regular calendar)

**Reply by Futsch1** [reply]
This would actually be (easily) possible, I will offer this as an alternative to the tabular view in the next release.
